index.html

```diff
@@ -20,16 +42,164 @@
+              <li class="skill-item">GitLab</li>
+              <li class="skill-item">Codepen</li>
+              <li class="skill-item">Terminal</li>
+            </ul>
+          </li>
+          <li class="skill" id="skill-three">
+            <h3 class="skill-title">Preprocessors</h3>
+            <ul class="sublist">
+              <li class="skill-item">Sass</li>
+              <li class="skill-item">Markdown</li>
+            </ul>
+          </li>
+        </ul>
+      </section>
+      <!-- Contact section -->
+      <section class="contact-links">
+        <a href="https://www.twitter.com/jsivahera" class="contact-link" id="twitter" title="Reach out on Twitter">
+          <i class="fab fa-twitter"></i>
+        </a>
+        <a href="https://www.linkedin.com/jsivahera" class="contact-link" id="linkedin" title="Reach out on LinkedIn">
+          <i class="fab fa-linkedin"></i>
+        </a>
+        <a href="https://www.github.com/jssol" class="contact-link" id="github" title="Reach out on Github">
+          <i class="fab fa-github"></i>
+        </a>
+        <a href="https://www.dribbble.com/jsivahera" class="contact-link" id="dribbble" title="Reach out on Dribble">
+          <i class="fab fa-dribbble" ></i>
+        </a>
+        <a href="https://www.facebook.com/jsivahera" class="contact-link" id="facebook" title="Reach out on Facebook">
+          <i class="fab fa-facebook" ></i>
+        </a>
+        <a href="https://www.codepen.com/jsivahera" class="contact-link" id="codepen" title="Reach out on Codepen">
+          <i class="fab fa-codepen" ></i>
+        </a>
+      </section>
+      <!-- Internal link to the projects -->
+      <a href="index.html#projects" id="to-projects">
+        <svg width="20" height="24" viewBox="0 0 19 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M0 12.8005L9.15 21.6505L18 12.8005" stroke="#C4C4C4"/>
+          <path d="M9.1499 0.500488V21.8005" stroke="#C4C4C4"/>
+        </svg>
+        <span>Work</span>
+      </a>
+      <!-- My projects section -->
+      <section class="projects-section">
+        <h2 class="projects-title">My recent work</h2>
+        <section class="projects">
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
```

Comment:
- [ ] Good job on your project. You are really doing great. However, I think the above `section` element and `p` tag can be replaced with `ul` and `li` elements respectively. That is a list of technologies after all. What do you think:thinking: ?